docs/devel/roadmap.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap
+
+The following table contains a priority-ordered list of work items and features the team
+plans to work on (as of March 2016).
+
+P0 items are considered blockers for next (`1.x`) release. All else is tackled on best effort basis.
+
+|Area                         |Item                   |Relative priority                 |
+|-----------------------------|-----------------------|-------------------------|
+|View cluster resources       |Next iteration of UX that scales for all Kubernetes resources and various use cases|P0|
+|View cluster resources       |Handle Services - list and basic details view|P0|
+|View cluster resources       |Handle Replica Sets - list and basic details view|P0|
+|View cluster resources       |Handle Pods - list and basic details view|P0|
+|View cluster resources       |Handle Deployments - list and basic details view|P0|
+|View cluster resources       |Handle Daemon Sets - list and basic details view|P0|
+|View cluster resources       |Handle Pet Sets - list and basic details view|P0|
+|View cluster resources       |UI is aware of namespaces/tenants|P0|
+|View cluster resources       |Handle Nodes - list and basic details view|P1|
+|View cluster resources       |Handle Jobs - list and basic details view|P1|
+|View cluster resources       |Show events for resources|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Nodes|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Containers|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Pods|P1|
+|Monitoring/troubleshooting   |Surface errors/warnings for all resources shown|P1|
+|Manage apps/resources        |YAML edit for resources displayed|P1|
+|Manage apps/resources        |Delete support for resources displayed|P1|
+|Create apps                  |Deploy form: migrate to Deployment objects|P1|
+|Multi cluster (Ubernetes)    |Support multiple clusters in one UI|P1|
+|Security/IAM                 |Make the UI securely accessible from outside of a cluster|P1|
+|Security/IAM                 |Make the UI act on behalf of a user|P1|
+|Security/IAM                 |IAM: don't crash/work correctly when IAM is enabled|P1|
```

Comment:
We'll definitely work on security as soon as we can. As stated, P0 tasks are blockers and they need to be completed before other tasks. Security is not a blocker but it is high on our list. You are more then welcome to help us if you want :)